.github/workflows/ci.yaml

```diff
@@ -0,0 +1,64 @@
+name: Continuous Integration
+on:
+  push:
+    branches:
+      - 'main'
+      - 'master'
+      - 'refs/heads/v[0-9]+.[0-9]+.[0-9]+'
+  pull_request:
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        title:
+          - "6.6.6"
+          - "6.6.6 pre-ALPHA"
+        description:
+          - "bla bla bla"
+          - ""
```

Comment:
@sc0ttkclark Anything you would add there? These are the ones that first came to mind after your PR.

Reply:
Looks good to me